src/ui/find-ui/src/FamilyHubs.ServiceDirectory.Web/Models/Service.cs

```diff
@@ -1,24 +1,18 @@
 ﻿
 namespace FamilyHubs.ServiceDirectory.Web.Models
 {
-    public enum ServiceType
-    {
-        FamilyHub,
-        Service
-    }
-
     //todo: type hierarchy, rather than type? or just null what we don't have?
     // when / opening hours will show regular schedule only. holiday schedule will be ignored for mvp (probably just show the description field)
     public sealed record Service(
-        ServiceType Type,
         string Name,
         //todo: what's actually mandatory?
         double? Distance,
         IEnumerable<string> Cost,
         IEnumerable<string> Where,
         IEnumerable<string> When,
         IEnumerable<string> Categories,
-        string? AgeRange = null,
+        IEnumerable<string> ServiceDeliveries,
+    string? AgeRange = null,
```

Comment:
nitpick: What's happened with the indentation?

Reply:
I have no idea :D but I've updated it